src/gluonts/mx/__init__.py

```diff
@@ -24,3 +24,27 @@
 Tensor = typing.Union[mx.nd.NDArray, mx.sym.Symbol]
 
 from . import prelude as _  # noqa
+from .trainer import Trainer  # noqa
+from .util import get_hybrid_forward_input_names, copy_parameters  # noqa
+from .kernels import RBFKernel  # noqa
+from .batchify import as_in_context, batchify  # noqa
+from .model.estimator import GluonEstimator  # noqa
+from .model.predictor import RepresentableBlockPredictor  # noqa
+from .distribution import DistributionOutput, GaussianOutput  # noqa
+from .block.scaler import MeanScaler, NOPScaler  # noqa
```

Comment:
We could sort the imports here (`isort` to the rescue!)

Reply:
Sorted ✔️